### DISCUSSION TITLE
GitHub Mobile Syntax Highlight

### DISCUSSION THREAD

**Discussion by Neodevils** [community/community#35893] [Mobile]
Hello 👋🏻

GitHub Mobile is syntax highlighting is a little bit darker than it should be. And syntax highlighting is seemed wrong to me. While strings showed as dark blue (which used for numbers, boolean and properties) and properties, boolean and numbers is vice versa.

#### GitHub Website's Syntax Highlight 

> ![IMG_20221012_125338](https://user-images.githubusercontent.com/71941576/195313877-cd13e61f-6097-4320-99ba-97d4f0a476e2.jpg)

#### GitHub Mobile's Syntax Highlight 

> ![IMG_20221012_130026](https://user-images.githubusercontent.com/71941576/195314451-c4a9caab-2fc3-4198-9b7b-db64f28d916d.jpg)

### Here is the list of GitHub Syntax Highlight Colors 
https://gist.github.com/Neodevils/ccb2e9a5c8092400d0d10f4fce151748

Hope that helps!

**Comment by iskae** [comment]
Hi @Neodevils 👋 

Thanks so much for reaching out ❤️  We already have an internal tracking item for this and I will be adding your feedback in there 👍 

Please stay tuned for future updates 😊 

Cheers!

**Reply by Neodevils** [reply]
I'll be waiting 👀.
